frontend/src/metabase-types/types/Database.ts

```diff
@@ -36,7 +35,8 @@ export type Database = {
   features: DatabaseFeature[];
   is_full_sync: boolean;
   is_sample: boolean;
-  native_permissions: DatabaseNativePermission;
+  native_permissions: DataAccessPermission;
+  data_model_permission?: DataAccessPermission;
```

Comment:
The same question - is it truly optional, or will the backend always set, e.g. to `none`? If it's optional, how is it different to `none` or `write`?